### PR TITLE
update examples for MyriaX doc

### PR DESCRIPTION
**Pull request by jingjingwang**

### CODE REVIEW

**Comment by jingjingwang** [comment]
http://myria.cs.washington.edu/docs/myriaX.html

**Comment by jingjingwang** [comment]
@dhalperi  I think you're the best person to be assigned since the issue was created by you :)

File: src/edu/washington/escience/myria/util/DeploymentUtils.java [hunk 1]
```diff
@@ -292,7 +293,10 @@ public static void ensureMasterStart(final String hostname, final int restPort,
           break;
         }
         // expected for the first few trials
-        LOGGER.warn("expected exception occurred", e);
```
Comment (dhalperi) [inline src/edu/washington/escience/myria/util/DeploymentUtils.java]: Should save the exception here and, if the master never comes up, print the last one we received. Sometimes the errors are useful!

Comment (dhalperi) [inline src/edu/washington/escience/myria/util/DeploymentUtils.java]: @jingjingwang bump -- this is why this PR is not yet merged.

**Comment by jingjingwang** [comment]
@dhalperi  done :)

**Pull request merged**